csharp/ql/lib/ext/System.IO.StreamReader.model.yml

```diff
@@ -0,0 +1,18 @@
+extensions:
+  - addsTo:
+      pack: codeql/csharp-all
+      extensible: extSummaryModel
+    data:
+      - ["System.IO", "StreamReader", False, "StreamReader", "",                                                                                "", "Argument[0]", "Argument[this]", "taint", "manual"]
```

Comment:
These models should be included in `System.IO.model.yml`.
The convention is that it is the namespace that decides which file the models should be added to (corresponding to the first component of the model).
Otherwise everything looks good!